src/popup/components/TransactionItem.vue

```diff
@@ -136,6 +140,11 @@ export default defineComponent({
           hash: props.transaction.hash,
           transactionOwner: props.transaction.transactionOwner,
         },
+        query: {
+          address: props.isMultisig
```

Comment:
Shall we maybe rename the query parameter to a `owner` for more clarity? It's just a suggestion.